modules/storages/app/models/queries/storages/project_storages/filter/storage_id_filter.rb

```diff
@@ -34,6 +34,10 @@ def type
       :list
     end
 
+    def human_name
+      ::Storages::ProjectStorage.human_attribute_name(name)
```

Comment:
Should we also add this method to the `modules/storages/app/models/queries/storages/work_packages/filter/storages_filter_mixin.rb` ?

Reply:
true, those probably also fail with an ugly 501 error, when receiving a wrong filter value. The `human_name` method is called on a filter, when the query tries to build error messages for them.